dc-cudami-editor/src/locales/de/translation.json

```
@@ -163,13 +163,15 @@
       "url": "Einbinden einer Datei aus dem Internet durch Eingabe einer gültigen URL"
     },
     "totalElements": {
-      "digitalObjects": "{{count}} Digitalisate gefunden",
-      "subcollections": "{{count}} Sammlungen gefunden"
+      "digitalObjects": "{{ count }} Digitalisat(e) gefunden",
+      "subcollections": "{{ count }} Sammlung(en) gefunden",
+      "webpages": "{{ count }} Webseite(n) gefunden"
```

Hier kannst du dir i18next's Plural-Support zunutze machen:

```suggestion
      "digitalObjects": "Digitalisat gefunden",
      "digitalObjects_plural": "{{ count }} Digitalisate gefunden",
      "subcollections": "Sammlung gefunden",
      "subcollections_plural": "{{ count }} Sammlungen gefunden",
      "webpages": "Webseite gefunden",
      "webpages_plural": "{{ count }} Webseiten gefunden"
```
https://www.i18next.com/translation-function/plurals

---

Danke, hab ich umgebaut :+1: